pkgs/top-level/aliases.nix

```diff
@@ -1575,6 +1575,8 @@ mapAliases {
   transmission-qt = lib.warn (transmission3Warning {suffix = "-qt";}) transmission_3-qt; # Added 2024-06-10
   treefmt = treefmt2; # 2024-06-28
   libtransmission = lib.warn (transmission3Warning {prefix = "lib";}) libtransmission_3; # Added 2024-06-10
+  tracker = lib.warn "tracker has been renamed to tinysparql" tinysparql; # Added 2024-09-30
+  tracker-miners = lib.warn "tracker-miners has been renamed to localsearch" localsearch; # Added 2024-09-30
```

Comment:
```suggestion
  tracker = tinysparql; # Added 2024-09-30
  tracker-miners = localsearch; # Added 2024-09-30
```

Reply:
You probably want to continue the discussion in https://github.com/NixOS/nixpkgs/pull/333911#discussion_r1777463411 :joy: